### DISCUSSION TITLE
Installation problem

### DISCUSSION THREAD

**Discussion by mpaglia0** [mosparo/mosparo#167] [Q&A] [answered]
Hi all,
I am trying to install Mosparo right now.
Created a directory ` mosparo`  in the ` /public`  dir of my own domain.
Downloaded ZIP, extracted it and copied all content in ` /public/mosparo`  via FTP.
Open the browser and point it to ` my_domain.it/mosparo` .
Receive error:
Forbidden
You don't have permission to access this resource.
Additionally, a 403 Forbidden error was encountered while trying to use an ErrorDocument to handle the request.

PHP version: 8.1.1
Requirements: all saisfied
SSL certificate: present and not expired

**Comment by zepich** [comment, marked as answer]
Hi @mpaglia0 

Thank you very much for your question.

I think you should open `https://my_domain.it/mosparo/public/` in your browser since there is no index file in the mosparo root directory (the .htaccess file should remap it to the `public` directory, but I'm not sure which server type you use (are you using Apache?).

But in general, we recommend installing mosparo in the root of a domain (`my_domain.it`) or on a subdomain (`mosparo.my_domain.it`), not like you did in a subdirectory. If you install it on a domain or subdomain, you can point the virtual host's document root to the `public` directory of mosparo.

Please let me know if that helps or if you still need help accessing mosparo.

Kind regards,

zepich

**Comment by mpaglia0** [comment]
Ciao @zepich 
First of all please acccept my apologize but I completely missed the subdomain information... :-(
Anyway I pointed my browser to `https://my_domain.it/mosparo/public/` and it went to `https://my_domain.it/mosparo/public/setup` but I obtained an Error 404
Created a `mosparo` subdomain and all was OK.
Installation super-easy and quick.
Now I will start try Mosparo and during Xmas holidays will start translating it in Italian.

Thanks!!
Maurizio

**Reply by zepich** [reply]
Hi @mpaglia0 

Thank you very much for your response.

Don't worry, I know how it is. Sometimes, we just don't see the simplest and most important information...

I'm happy that it works now. If you have any other questions, please let me know. I'm looking forward to your translations!

Have a good festive time!

Kind regards,

zepich
